src/Elrond/NetworkConfig.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "NetworkConfig.h"
+
+using namespace TW;
+using namespace TW::Elrond;
+
+NetworkConfig::NetworkConfig() :
+    chainId("1"), // Mainnet
+    gasPerDataByte(1500),
+    minGasLimit(50000),
+    minGasPrice(1000000000),
+    gasCostESDTTransfer(200000),
+    gasCostESDTNFTTransfer(200000) {
+}
+
+void NetworkConfig::setChainId(const std::string& value) {
+    this->chainId = value;
+}
+
+void NetworkConfig::setGasPerDataByte(uint32_t value) {
```

Comment:
minor suggestion: place the setter and the getter for each field one near each other?

Reply:
Fixed.